source/accnt_mgmt.rst

```diff
@@ -116,10 +116,10 @@ The following request:
   .. literalinclude:: code/python/account_management/create_cloud_dir_req.py
       :language: python
 
-Would yield the following response:
-
 .. only:: rest
 
+  Would yield the following response:
```

Comment:
In a number of places, I moved text like "the following response" into only:: rest, because I wanted to use different language about the response (it doesn't really make sense to show it in my case).